docs/Using A2C.md

```diff
@@ -0,0 +1,45 @@
+Example code to run the algo
```

Comment:
Make this a heading and rename it to something more descriptive like "Using A2C on CartPole-v0"